kuksa-val-server/src/VSSTypeSanitizer.cpp

```diff
@@ -165,8 +165,23 @@ void VssDatabase::checkAndSanitizeType(jsoncons::json &meta, jsoncons::json &val
         checkBoolType(val);
     }
     else if (dt == "string") { 
+      if (meta.contains("allowed") ) {
```

Comment:
I would not use those complicated vss_xx_supported constants, I would just silently support both, ie. sth lik

```
if (meta.contains("allowed") {
  checkEnumType(meta["allowed"], val);
}
else if (meta.contains("enum") ) {
checkEnumType(meta["enum"], val);
}
```